ru/lessons/basics/pattern-matching.md

```diff
@@ -12,7 +12,7 @@ lang: ru
 
 ## Оператор сопоставления
 
-В языке Elixir оператор `=` называется оператором сопоставления. С его помощью можно как назначать, так и сравнивать значения:
+В языке Elixir оператор `=` на самом деле является оператором сопоставления по аналогии со знаком равенства в алгебре. Его использование превращает всё выражение в уравнение, и Elixir сопоставляет левую часть выражения с правой. В случае успешного сопоставления вернётся решение уравнения, иначе возникнет ошибка:
```

Comment:
And if you're talking about example like

`{true, x} = Application.getenv()`
the solution is `x`, while `{true, x}` gets returned.

@nscyclone